src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @include animation-hover(transform, scale(1.2));
+
+  &__images {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
```

Comment:
The width for `&__images` should be defined using the maximum width (`max-width`) property instead of `width` to make the image more responsive. This ensures that the image can scale down if necessary, without exceeding the specified maximum width.